src/web/client/WebExtensionContext.ts

```diff
@@ -272,8 +272,8 @@ class WebExtensionContext implements IWebExtensionContext {
         const schema = queryParamsMap.get(schemaKey.SCHEMA_VERSION) as string;
         // Initialize context from URL params
         this._currentSchemaVersion = schema;
-        this._defaultEntityType = entityName.toLowerCase();
-        this._defaultEntityId = entityId;
+        this._defaultEntityType = entityName ? entityName.toLowerCase() : queryParamsMap.has(Constants.queryParameters.ENTITY) ? queryParamsMap.get(Constants.queryParameters.ENTITY) as string : "";
```

Comment:
can be refactored to enitityId ?? queryParamsMap.get(Constants.queryParameters.ENTITY_ID) ?? ""